Nim/Nim/Nim.cpp

```diff
@@ -0,0 +1,133 @@
+using namespace std;
+
+#include <iostream>
+#include <string>
```

Comment:
We were not supposed to use the `string`-library here, yet, but it's not an issue :)